modules/serverless-kafka-tls-sasl.adoc

```diff
@@ -0,0 +1,87 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+= Configuring authentication for Kafka
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure the `KafkaChannel` to work against a protected Kafka cluster using TLS or SASL.
```

Comment:
`KafkaChannel` type markup should be followed by "object" or "resource", or written as "Kafka channel", as per the new OCP docs guidelines